README.md

```diff
@@ -76,7 +76,7 @@ run the following:
 pip install tensorflow-recommenders-addons
 ```
 
-By default, CPU version will be installed. To install GPU version, run the following:
+Before the release of combined CPU and GPU version, to install GPU version, run the following:
```

Comment:
Better to explain more clearly, for example, the specific version(0.7.x) after which we make this change.

Reply:
updated